### PR TITLE
Launch week 5 landing

### PR DESCRIPTION
**Pull request by MildTomato**
Landing page for launch week 5 🚀

### CODE REVIEW

**Comment by MildTomato** [comment]
just need to fix some rewrites

File: packages/config/ui.config.js [hunk 1]
```diff
@@ -289,6 +287,32 @@ const uiConfig = {
           border: '1px solid hsla(0, 0%, 39.2%, 0.2)',
           borderRadius: '3px',
         },
+        '.no-scrollbar': {
```
Comment (MildTomato) [inline packages/config/ui.config.js:290]: just updates from new UI lib updates

File: packages/config/ui.config.js [hunk 2]
```diff
@@ -324,12 +348,12 @@ const uiConfig = {
             bordershadow: (value) => {
               return {
                 boxShadow: `
-                var(--colors-blacka1) 0px 0px 0px 0px, 
-                var(--colors-blacka1) 0px 0px 0px 0px, 
-                var(--colors-blacka8) 0px 1px 1px 0px, 
+                var(--colors-blackA1) 0px 0px 0px 0px,
```
Comment (MildTomato) [inline packages/config/ui.config.js:351]: fixing UI lib alpha issies

**Review by MildTomato — COMMENTED** [review]
comments to explain extra things

File: package.json [hunk 3]
```diff
@@ -63,6 +63,7 @@
   ],
   "dependencies": {
     "@mdx-js/react": "^2.1.1",
-    "mini-svg-data-uri": "^1.4.4"
+    "mini-svg-data-uri": "^1.4.4",
```
Comment (MildTomato) [inline package.json:66]: removing...

**Review by MildTomato — COMMENTED** [review]
removing

File: apps/www/components/LaunchWeek/HackathonSection.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+              </Button>
+            </Link>
+          </div>
+        </div>
+
+        <div className="grid grid-cols-2 gap-16">
+          <div className="flex flex-col gap-3">
+            <h3 className="text-scale-1200 text-lg">Prizes</h3>
+            <p className="text-scale-1100 text-sm">
+              There are 5 categories to win. There will be winner prize and a runner-up prize in
```
Comment (dannykng) [inline apps/www/components/LaunchWeek/HackathonSection.tsx:39]:
```suggestion
              There are 5 categories to win. There will be a prize for the winner and a runner-up prize in
```

File: apps/www/components/LaunchWeek/ScheduleInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      <div className="dark:bg-scale-300 flex flex-col gap-4 overflow-hidden rounded-md border bg-white shadow-sm md:max-w-lg">
+        <div className="flex flex-col gap-3 p-10 pb-0">
+          <h3 className="text-scale-1200 text-lg">You can still win a lucky gold ticket</h3>
+          <p className="text-scale-1100 text-sm">
+            A few of the lucky attendees for Launch Week will get a limited edition Supabase goodie
+            bag.
+          </p>
+        </div>
+        <div className="px-10">
+          <Button type="default">Get a ticket</Button>
```
Comment (dannykng) [inline apps/www/components/LaunchWeek/ScheduleInfo.tsx]: This doesn't link to anything

**Review by dannykng — COMMENTED** [review]
I think we should also add AM/PM to the times (e.g., it's not clear when the hackathon ends).

File: apps/www/components/LaunchWeek/ScheduleInfo.tsx (hunk 5 above)
Comment (dannykng) [reply apps/www/components/LaunchWeek/ScheduleInfo.tsx]: Ah you got it 😅

**Comment by MildTomato** [comment]
> I think we should also add AM/PM to the times (e.g., it's not clear when the hackathon ends).

Changed it to 23:59 - is that ok??

**Review by dannykng — APPROVED** [review]

**Pull request merged**